docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[configure-endpoint-integration-policy]]
 = Configure the {endpoint-sec} integration policy
 
-After the {agent} is installed with the {endpoint-sec} integration, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum or Enterprise license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, and trusted applications to meet your organization's security needs.
+After the {agent} is installed with the {endpoint-sec} integration, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum or Enterprise license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, trusted applications, event filters, and host isolation exceptions to meet your organization's security needs.
 
 NOTE: You must have admin permissions in {kib} to configure an integration policy.
```

Comment:
We may want to clarify this; this section implies the user will be editing the policy in the security application, which currently requires _superuser_.

Configuring integrations within Fleet requires Fleet Admin

Reply:
@caitlinbetz Do you think we need to mention the Fleet requirement here? This page only covers configuring a policy from the POV of the Security app, so mentioning Fleet permissions here might be distracting. We do cover installing and configuring via Fleet/Integrations [elsewhere](https://www.elastic.co/guide/en/security/current/install-endpoint.html).
```suggestion
NOTE: You must have the {kib} `superuser` role to configure an integration policy in the {security-app}.
```